docs/content/en/docs/osmgmt/artifacts.md

```diff
@@ -1136,6 +1136,13 @@ You can now run the `image-builder` CLI with the `files-config` option, with thi
      "no_proxy": "<optional comma seperated list of domains that should be excluded from proxying>"
   }
   ```
+In a proxy-enabled environment, `image-builder` uses `wget` to download artifacts instead of `curl`, as `curl` doesnt support reading proxy environment variables. In order to add `wget` to the node OS, add the following to the above json configuration file
```

Comment:
```suggestion
In a proxy-enabled environment, `image-builder` uses `wget` to download artifacts instead of `curl`, as `curl` does not support reading proxy environment variables. In order to add `wget` to the node OS, add the following to the above json configuration file:
```